src/components/LeftDrawer/LeftDrawer.js

```diff
@@ -9,29 +9,30 @@ import Divider from '@material-ui/core/Divider';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import {
-  toggleLeftDrawer, toggleRightDrawer, setTabValue, selectLeftMenuItem,
```

Comment:
No need to toggle the right drawer or set the tab value here anymore?

Reply:
We used to have a custom value in the list that would open the right drawer; I got rid of that for now and just added the list of "must see" features. If we want to add the "Event" list icon to open the right menu later on we can